src/main/scala/com/github/ehiggs/spark/terasort/TeraSort.scala

```diff
@@ -59,8 +59,8 @@ object TeraSort {
     val sc = new SparkContext(conf)
 
     if (splitSize != "-1") {
-      sc.hadoopConfiguration.set("mapred.min.split.size", splitSize)
-      sc.hadoopConfiguration.set("mapred.max.split.size", splitSize)
+      sc.hadoopConfiguration.set("mapreduce.input.fileinputformat.split.minsize", splitSize)
```

Comment:
Is this because the `mapred.` configs are deprecated?

Reply:
Yup. See https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/DeprecatedProperties.html